### ISSUE TITLE
[BUILD] Take into consideration `CARGO_TARGET_DIR`

### ISSUE THREAD

**Issue by henil** [Morganamilo/paru#89]
If the user has `CARGO_TARGET_DIR` variable set, it gives this error:
> install: cannot stat 'target/release/paru': No such file or directory

Changing line 33 in `PKGBUILD` to this seems to fix it for me.
```
install -Dm755 ${CARGO_TARGET_DIR:-./target}/release/paru "${pkgdir}/usr/bin/paru"
```

**Comment by Morganamilo** [comment]
I guess that would fix it. What about using cargo install instead? Or getting cargo to tell us where the binary is?

**Comment by henil** [comment]
Tried to do `cargo install paru` but got this error:
```log
error[E0277]: the trait bound `std::vec::Vec<u8>: std::convert::From<&std::string::String>` is not satisfied
  --> /home/panda/.local/share/cargo/registry/src/github.com-1ecc6299db9ec823/alpm-utils-0.5.0/src/db.rs:52:26
   |
52 |             let pkg = db.pkg(&pkg);
   |                          ^^^ the trait `std::convert::From<&std::string::String>` is not implemented for `std::vec::Vec<u8>`
   |
   = help: the following implementations were found:
             <std::vec::Vec<T> as std::convert::From<&[T]>>
             <std::vec::Vec<T> as std::convert::From<&mut [T]>>
             <std::vec::Vec<T> as std::convert::From<[T; N]>>
             <std::vec::Vec<T> as std::convert::From<std::borrow::Cow<'a, [T]>>>
           and 6 others
   = note: required because of the requirements on the impl of `std::convert::Into<std::vec::Vec<u8>>` for `&std::string::String`
```
> Or getting cargo to tell us where the binary is?

I don't know of any way to find it. Is there a cargo plugin for it?

**Comment by Morganamilo** [comment]
Fixed in v1.1.0
